.changeset/shaggy-mice-jump.md

```diff
@@ -0,0 +1,14 @@
+---
+"thirdweb": minor
+---
+
+Show in-app wallet users an option to export their private key
+
+### Usage
+
+```tsx
+import { inAppWallet } from "thirdweb/wallets";
+const wallet = inAppWallet({
+  showPrivateKeyExport: true,
```

Comment:
It's a bit odd to out this option here since it only applies to the UI components.

Using in app wallet with your own UI would make this option do nothing at all.

I know we have the auth options in there too which also don't really make sense there. Feels like we should move those options to the connect button/embed itself. Would only apply is inAppWallet is in the wallet list, but def feels more appropriate there. Wdyt?

Reply:
I don't love the idea of adding it to the connect button or embed props given there are already so many of those and they're very hard to discover/understand. At the same time, I agree it's a bit odd here for someone not using the UI.

Maybe we should enable it in our UI and not provide the option to disable it? If someone wants to change our defaults, they should build their own UI (and we should provide more low-level components to make that easier).